go.mod

```diff
@@ -1,8 +1,6 @@
 module github.com/redhat-developer/rhdh-operator
 
-go 1.22.7
-
-toolchain go1.22.9
+go 1.23
```

Comment:
we don't need a toolchain modifier and/or .z suffix versions anymore?

Reply:
That was the outcome of `go mod tidy`. I assume it was removed because Go determined that it was redundant.
Since we are building already with Go 1.23, it should use its own bundled toolchain. But you're right, I think it would be safer to specify it.